### DISCUSSION TITLE
Transcription size

### DISCUSSION THREAD

**Discussion by LeninSG21** [deepgram/community#304] [General help] [answered]
Hi! I'm implementing Deepgram for the first time with a webhook callback and ran into the `[Error: request entity too large]` issue. I understand that by changing the limit in my json parser in express I can accept larger requests, and I have it working now. However, I wanted to know the size of a one-hour transcription with **smart format** and **diarization** to better adjust the limit on my server. 

Thanks for the awesome work!

**Comment by jkroll-deepgram** [comment, marked as answer]
Hi @LeninSG21, I grabbed [this lecture](https://www.youtube.com/watch?v=Hhobq4fs87w) as a 1-hour audio source, and ran it through Deepgram with `tier=nova&smart_format=true&diarize=true`. The response was 2.7 MB. Note that the exact size will vary based on the number of speakers and speed of conversation, but it's an approximate estimate. Does that line up with what you've been observing in your transcripts so far?

**Reply by LeninSG21** [reply]
Thanks @jkroll-deepgram! I've been looking at smaller transcriptions. Usually a 4 minute transcriptions has a ~80kB size, so I expected a 1 hour transcription to be less than 2MB, but still wanted to see if other people with longer transcriptions has a more accurate size.
